include/mbedtls/x509_crt.h

```diff
@@ -331,12 +331,22 @@ typedef void mbedtls_x509_crt_restart_ctx;
 /**
  * Default security profile. Should provide a good balance between security
  * and compatibility with current deployments.
+ *
+ * This profile permits:
+ * - SHA2 hashes with at least 256 bits: SHA-256, SHA-384, SHA-512.
+ * - Elliptic curves with 255 bits and above.
```

Comment:
224-bit curves are excluded. [Even in 2016](https://eprint.iacr.org/2018/298.pdf), support for secp224r1 was negligible compared to secp256r1, so I don't think we're losing any interoperability by disabling secp224r1.